pkg/templates/toolchaincluster/template_test.go

```diff
@@ -0,0 +1,28 @@
+package toolchaincluster
+
+import (
+	"testing"
+
+	"github.com/codeready-toolchain/host-operator/deploy"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoadObjectsFromTemplates(t *testing.T) {
```

Comment:
It would be nice to have some sanity check that all variables were properly set. 
If you move the function into toolchain-common, then you could use some dummy template and check the whole content of the decoded manifests.

Reply:
Yes I can add some more tests using dummy templates once it's in toolchain-common.

Reply:
I've moved the functionality here: https://github.com/codeready-toolchain/toolchain-common/pull/365 as is. I've only renamed the function and added more tests. 

I'll remove the code from this PR, and leave only the templates.